docs/adr/013-add-provisioning-saga.md

```diff
@@ -0,0 +1,72 @@
+# 13. Add Provisioning Saga Workflow
+
+Authors: Adi Abramovich
+
+## Status
+
+Proposed
+
+## Problem Statement
+
+This ADR exists because there is a need of making the build and launch workflow more user-friendly. 
+this ADR will include three propositions regarding how should we implement this feature and what should we take into consideration.
+
+## Goals
+
+* Make the saga workflow fast, secure, and simple
+* Choose one of the architectures prop3os3ed
+
+## Non-goals
+
+## Current Architecture
+
+* Currently the Image builder wizard and the Provisioning Wizard are not connected.
+After collecting all the information needed in order to create an image The image builder wizard closes and we are redireced to Image builder main page which lists all the images that already have been created.
+Only after the image was created successfully, a **"Launch"** button appears which leads to the Provisioning wizard.
+
+## Proposed Architecture
+
+I have thought about three architecture options where all three options have the first steps in common:
+
+* Image builder collects all the information needed to create an image on a specific hyper-scaler
+* Pressing the **Build and launch** button on the wizard
+* Validate the build manifest
+* A background job is running in order to create an image
+* The Provisioning wizard opens up
+
+Important information to remember:
+
+* The Provisioning is dependent on the image id in order to continue and finish the launch
+* The shared sources are given directly from the image variable from IB Wizard
+
+**In the following section I will elaborate on the differences:**
+
+1. Combining Image Builder wizard with the Provisioning wizard and showing the status of the instance on Image Builder main page:
```

Comment:
I'm not clear what's first, when do we poll and where (the Prov UI, prov backend - background job, IB?)